### PR TITLE
creating crawler for Brazil Central Bank QGP

### PR DESCRIPTION
**Pull request by bgmello**
This PR creates the crawler for the  Brazil Central Bank QGP describe below:

The QGP, managed by the Central Bank of Brazil, is essential for regulatory 
  compliance in the financial industry. It details names of persons and entities
  facing temporary bans from offering auditing services to Central Bank-regulated institutions.
  These restrictions, often a result of penalties in administrative proceedings,
  aim to ensure the reliability and integrity of those in key auditing roles.
  The QGP reflects the Central Bank's dedication to maintaining
  stability and transparency in Brazil's financial system.

Issue https://github.com/opensanctions/crawlers/issues/16

### CODE REVIEW

File: datasets/br/bcb_prohibited/crawler.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    # It cannot be anything other than those two
+    else:
+        context.log.error("Tax number is neither a CPF nor CNPJ")
+
+    entity.id = context.make_id(tax_number)
+    entity.add("name", input_dict["Nome"])
+    entity.add("taxNumber", tax_number)
+    entity.add("country", "br")
+    entity.add('topics', 'corp.disqual')
```
Comment (jbothma) [inline datasets/br/bcb_prohibited/crawler.py:48]: For this list I think debarment is most appropriate

```suggestion
    entity.add('topics', 'debarment')
```

File: datasets/br/bcb_prohibited/crawler.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    if len(tax_number) == 14:
+        entity.add_schema("Company")
+
+    # If it's 11 digits, then it is a CPF: https://en.wikipedia.org/wiki/CPF_number
+    elif len(tax_number) == 11:
+        entity.add_schema("Person")
+
+    # It cannot be anything other than those two
+    else:
+        context.log.error("Tax number is neither a CPF nor CNPJ")
```
Comment (jbothma) [inline datasets/br/bcb_prohibited/crawler.py:42]: very nice

File: datasets/br/bcb_prohibited/crawler.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+                        - Prazo_em_ano
+                        - Inicio_do_cumprimento
+                        - Prazo_final_penalidade
+    :param entity: Entity the sanction refers to.
+    :param context: The context object.
+    """
+
+    sanction = h.make_sanction(context, entity)
+    sanction.add(
+        "program", "Brazil's Central Bank General Register of Persons Prohibited from Offering Auditing Services"
```
Comment (jbothma) [inline datasets/br/bcb_prohibited/crawler.py:69]:
```suggestion
        "program", "Brazil's Central Bank General Register of Persons and Companies Prohibited from Offering Auditing Services"
```

**Review by jbothma — CHANGES_REQUESTED** [review]
Thanks, looks great!

I think all the comments to https://github.com/opensanctions/opensanctions/pull/519 also apply here - for the entity id, just make the prefix br-cnpj when it's companies.

**Comment by bgmello** [comment]
Waiting for the PR #519 to be resolved so I can apply the same modifications here.

**Comment by bgmello** [comment]
I made the same changes outlined on #519

**Pull request merged**